schemas/csdl.schema.json

```diff
@@ -251,7 +251,7 @@
       "type": "object",
       "additionalProperties": false,
       "patternProperties": {
-        "@": {
+        "^@": {
```

Comment:
But a complex type can contain a `property@an.no.tation`.

Reply:
No, in CSDL JSON the value of a property _definition_ is an object, so "metadata" annotations on the property definition go into that object: `property:{@an.no.tation:...}`.